profiler/profile_test.go

```diff
@@ -183,6 +195,7 @@ main;bar 0 0 8 16
 		}
 
 		p, err := unstartedProfiler()
+		require.NoError(t, err)
```

Comment:
Unrelated change, can move it into a separate PR if y'all want.

Reply:
No need, this is welcome 🙂